src/Core/DefaultPolicy/TicTacToeDefaultPolicy.cpp

```diff
@@ -8,7 +9,7 @@ Reward TicTacToeDefaultPolicy::defaultPolicy(State state) {
     States childStates = this->getValidChildStates(state);
     States validChildStates;
     // TODO check the init of tempstate
-    State tempState(NULL);
+    State tempState(std::nullopt);
```

Comment:
Oh, didn't know we could do this. boost::any is pretty nice I guess

Reply:
We don't use boost::any anymore ;)

Reply:
Actually I will just remove those from CMake